### ISSUE TITLE
How to use S-Bert as a component ?

### ISSUE THREAD

**Issue by Zibo-Zhao** [UKPLab/sentence-transformers#1035]
Hi, thank you for the great works! 
I want to add S-Bert as a component of my model and finetune it jointly with the remaining architecture (using S-Bert producing sentence embeddings and use these embeddings as the inputs of remaining architecture). For example, each time a batch of sentences will be sent to S-Bert, and aftering obtaining the corresponding embeddings, I want to use them as the inputs of another multihead attentions module. After obtaining final loss I want to adjust the weights of S-Bert jointly with the remaining model. Can I achieve this purpose using this repo? or are there any easy way to this end (I didn't find related examples)?

**Comment by nreimers** [comment]
The SentenceTransformer model is a standard PyTorch module, where you can call the forward method on.

Here an example:
https://github.com/UKPLab/sentence-transformers/pull/765#issuecomment-781701188
